test/integration/bootstrap.sh

```diff
@@ -1,22 +1,22 @@
 #!/bin/sh
-test ! -e /tmp/folder && \
```

Comment:
I wonder if we want to start this file with a quick check for the existence of `TMPDIR`, defaulting it to `/tmp/` if it doesn't otherwise exist.

Reply:
Added changes